src/pages/impact/index.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from "react";
+import { Helmet } from "react-helmet";
+
+import ArticleSpotlightCard from "../../components/grid-aware/ArticleSpotlightCard";
+import Spacer from "../../components/grid-aware/Spacer";
+import StatsBarBlock from "../../components/grid-aware/StatsBarBlock";
+import StatsBlock from "../../components/grid-aware/StatsBlock";
+import Layout from "../../components/layout";
+import articleSpotlightImage from "../images/mission-hotel.jpeg";
+
+export default () => (
+  <Layout>
+    <Helmet>
+      <title>Making an impact</title>
+    </Helmet>
+    <StatsBlock
+      title="SF Service Guide"
+      subtitle="A platform designed to connect at-risk and homeless individuals to critical human services in San Francisco. Learn more about this program."
+      statCards={[
+        {
+          number: "3k",
+          secondaryNumber: 6,
+          statement: "Monthly users",
+        },
+        {
+          number: "13.4k",
+          secondaryNumber: 6,
+          statement: "Annual users",
+        },
+        {
+          number: "111",
+          statement: "New resources",
+        },
```

Comment:
I just noticed that the vertical alignment is a bit off on this card, since it doesn't have the secondary number. In the Figma design, it looks like everything should be bottom-aligned, similar to the stats bar block. Do you think you could make that change to the component? It could even be in the same PR as when you update the font styles for https://github.com/ShelterTechSF/sheltertech.org/issues/324.